tests/containers/podman_netavark.pm

```diff
@@ -198,6 +203,11 @@ sub run {
 
     remove_subtest_setup;
 
+    # restore the default ipv6 route
+    if ($ipv6_gateway && $ipv6_interface) {
+        assert_script_run("ip -6 route add default via $ipv6_gateway dev $ipv6_interface");
```

Comment:
We should think of a scenario the test code might not reach this part in case of a different failure. It would be nice to have a check and route restore in the post_fail_hook as well. Not all the backends create snapshots of the SUT.

Reply:
Fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20504/commits/1f6b2d21199f2116a0409d9af38ae65d42562683 

New VR (with intentional failure) http://rmarliere-openqa.qe.prg2.suse.org/tests/157#step/podman_netavark/241